src/components/EditActivityPage/EditActivityPage.js

```diff
@@ -34,66 +35,72 @@ const EditActivityPage = ({
 
   const activityId = get(editedActivity, 'id', id);
 
-  const asyncEditActivity = useCallback(
+  const useToast = () => {
```

Comment:
this is what I thought, but this hook should be in separate file. Probably you will use it many places.